datarepo-clienttests/src/main/java/testscripts/SoftDeleteDataset.java

```diff
@@ -80,25 +80,15 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
     BulkLoadArrayResultModel bulkLoadArrayResultModel =
         DataRepoUtils.expectJobSuccess(
             repositoryApi, ingestFileJobResponse, BulkLoadArrayResultModel.class);
-    String fileId = bulkLoadArrayResultModel.getLoadFileResults().get(0).getFileId();
-
-    // ingest the tabular data from the JSON file we just generated
-    // generate a JSON file with the fileref
-    String jsonLine =
-        "{\"VCF_File_Name\":\"name1\", \"Description\":\"description1\", \"VCF_File_Ref\":\""
-            + fileId
-            + "\"}\n";
-    byte[] fileRefBytes = jsonLine.getBytes(StandardCharsets.UTF_8);
+
     String jsonFileName = FileUtils.randomizeName("this-better-pass") + ".json";
     String dirInCloud = "scratch/softDel";
     String fileRefName = dirInCloud + "/" + jsonFileName;
-    String gsPath = FileUtils.createGsPath(fileRefBytes, fileRefName, testConfigGetIngestbucket);
 
     IngestRequestModel ingestRequest =
-        new IngestRequestModel()
-            .format(IngestRequestModel.FormatEnum.JSON)
-            .table("vcf_file")
-            .path(gsPath);
+        BulkLoadUtils.makeIngestRequestFromLoadArray(
+            bulkLoadArrayResultModel, testConfigGetIngestbucket, fileRefName);
+
```

Comment:
I'm glad this got a small lift--I will def be borrowing it!

Reply:
It saved me a lot of time to have that code already written!